src/stravalib/field_conversions.py

```diff
@@ -48,9 +64,26 @@ def enum_value(v: ActivityType | SportType) -> str:
 
 @optional_input
 def enum_values(enums: Sequence[ActivityType | SportType]) -> list[str | None]:
+    """A function that processes multiple enums and returns a list.
```

Comment:
This could be kept (or removed) but i haven't had time to dig into whether pydantic 2.x handles enum lists now.

Reply:
Same as above, let's not try to force backward compatibility by converting the list of enums to a list of str.